### PR TITLE
#DP-102: (Feat) Integration setup

### PR DESCRIPTION
**Pull request by feyton**
### PR Description
This PR setup the integration of the project. This will allow users to save the data in our backend database while also having the ability to read and write to the database

### Description of tasks that were expected to be completed

- [x] The app is able to establish the communication with the backend
- [x] Initial call has been made (Login)

### How has this been tested?
To test feature, 

1. Head to this **[ATLP BN](https://atlp-bn.herokuapp.com/)** version and create a new user
2. Using the credentials you got, login to this platform. The error will be displayed if the credentials are not valid

For quick testing use the following credentials

```
admin: {
email: admin@pulse.com
password: Andela123
}

coordinator: {
email: coordinator@pulse.com
password: Andela123
}

// You can always create a custom user

```

### Screenshots
image

### CODE REVIEW

**Comment by feyton** [comment]
I deployed a review app for you to try out this pull request 👇🏽

Heroku App: https://devpulse-review-app-77.herokuapp.com/


<hidden purpose="for-rewritable-pr-comment-action-use" value="Deploy"></hidden>

**Comment by Procech20** [comment]
The coverage has dropped by 0.9, you can try to handle that

**Review by Procech20 — APPROVED** [review]
The credentials are working, LGTM @feyton

**Review by samyshyaka — APPROVED** [review]

File: src/pages/Organization/AdminLogin.tsx [hunk 1]
```diff
@@ -22,24 +25,29 @@ function AdminLogin() {
...
+      toast.success(`Welcome`);
+      if (state) {
+        navigate(`${state}`);
+      } else {
+        navigate('/dashboard/');
+      }
+      return;
+    } catch (error) {
+      setError('password', { type: 'custom', message: 'Invalid credentials' });
+      setError('email', { type: 'custom', message: 'Invalid credentials' });
```
Comment (rukundo-kevin) [inline src/pages/Organization/AdminLogin.tsx:49]: Can you translate this error message.

**Review by rukundo-kevin — APPROVED** [review]
Great job @feyton, Just translate the error messages only.

**Review by ceelogre — COMMENTED** [review]
Resolve the conflicts so we can merge.

File: src/pages/Organization/AdminLogin.tsx (hunk 1 above)
Comment (feyton) [reply src/pages/Organization/AdminLogin.tsx]: The transltions has been added

**Review by Dev-nkundineza — APPROVED** [review]
Good work.

**Comment by feyton** [comment]
> 

The coverage has been raised

**Review by jeandamascyifuzo — APPROVED** [review]
Well done @feyton

**Pull request merged**